printing/printer.ml

```diff
@@ -905,7 +905,7 @@ let pr_assumptionset env sigma s =
     let pr_axiom env ax typ =
       match ax with
       | Constant kn ->
-          safe_pr_constant env kn ++ safe_pr_ltype env sigma typ
+          safe_pr_constant env kn ++ cut() ++ safe_pr_ltype env sigma typ
```

Comment:
Marks like `cut` or `spc` take sense in the contex of a surrounding box which can be `hov`, `hv`, `h`, `v`, ... See OCaml's [format](https://caml.inria.fr/pub/docs/manual-ocaml/libref/Format.html) for details.

Here, we may want an indentation of 2, like for the items after (`Positive`, `Guarded`, `TypeInType`). There is already a space in `safe_pr_ltype`, so, we tell to indent of an extra 1 if a cut is needed.
```suggestion
          hov 1 (safe_pr_constant env kn ++ cut() ++ safe_pr_ltype env sigma typ)
```

Note that the boxing can greatly be improved in many places of `printer.ml` actually.

Reply:
Thanks, that looks pretty OK imo